src/interfaces/README.md

```diff
@@ -1,17 +1,39 @@
-# Internal c++ interfaces
+# Internal c++ [`src/interfaces/`](../interfaces)
 
-The following interfaces are defined here:
+### The following interfaces are defined here:
 
 * [`Chain`](chain.h) — used by wallet to access blockchain and mempool state. Added in [#14437](https://github.com/bitcoin/bitcoin/pull/14437), [#14711](https://github.com/bitcoin/bitcoin/pull/14711), [#15288](https://github.com/bitcoin/bitcoin/pull/15288), and [#10973](https://github.com/bitcoin/bitcoin/pull/10973).
+	- Refactored in [#20494](https://github.com/bitcoin/bitcoin/pull/20494/commits)
 
 * [`ChainClient`](chain.h) — used by node to start & stop `Chain` clients. Added in [#14437](https://github.com/bitcoin/bitcoin/pull/14437).
 
 * [`Node`](node.h) — used by GUI to start & stop bitcoin node. Added in [#10244](https://github.com/bitcoin/bitcoin/pull/10244).
+	- Refactored in [#20494](https://github.com/bitcoin/bitcoin/pull/20494/commits)
 
 * [`Wallet`](wallet.h) — used by GUI to access wallets. Added in [#10244](https://github.com/bitcoin/bitcoin/pull/10244).
+	- Refactored in [#20494](https://github.com/bitcoin/bitcoin/pull/20494/commits)
```

Comment:
Perhaps drop the `/commits` path in the URL and add this line in one place only, referring to chain/node/wallet.